.gitignore

```diff
@@ -0,0 +1,329 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
```

Comment:
Could you also copy over `.editorconfig` from Castle Core.

Reply:
Done.